panoramapublic/src/org/labkey/panoramapublic/view/publish/confirmPublish.jsp

```diff
@@ -32,11 +33,8 @@
         var items = [];
         if (<%=!bean.isPublic()%>) {
             let html = 'Data at ' + <%=qh(bean.getAccessUrl())%> + ' will be made public.';
-            if (<%= bean.getLicense() != null %>) {
-                html += ' It will be available under the ' + <%=qh(bean.getLicense().getDisplayName())%> + ' license.';
-            }
+            html += <%= bean.getLicense() != null ? qh("It will be available under the " + bean.getLicense().getDisplayName() + ".") : HtmlString.EMPTY_STRING %>;
```

Comment:
Maybe I'm misreading the code, but if the license is null, I think this will render as:

`html += ;`

which isn't valid JS.

Reply:
Indeed.  It has been fixed. Thanks for catching.